### PR TITLE
fix: solved issue twin valkyr's ball

### PR DESCRIPTION
**Pull request by meowcpp**
## Changes Proposed:
-  Before this commit, twin valkyr's balls appeared only once during encounter
-  Balls should appear periodically

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4903

## Tests Performed:

- Build wihout errors
- Tested ingame - balls appeared periodically during whole encounter
- Tested on Win 10

## How to Test the Changes:

1. Teleport into toc **.tele crusader** and go inside
2. Defeat all bosses up to valkyrs
3. Start encounter - balls should appear periodically (not just once)

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_twin_valkyr.cpp [hunk 1]
```diff
@@ -275,7 +275,9 @@ struct boss_twin_valkyrAI : public ScriptedAI
         if( me->HasUnitState(UNIT_STATE_CASTING) )
             return;
 
-        switch( events.ExecuteEvent() )
+        uint8 eventId = events.ExecuteEvent();
+
+        switch( eventId )
```
Comment (Kitzunu) [inline src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_twin_valkyr.cpp:280]: https://www.azerothcore.org/wiki/cpp-code-standards#whitespace

Comment (meowcpp) [reply src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_twin_valkyr.cpp:280]: Actually whole script is written that way I've just changed events.ExecuteEvent() to evenId, normally I use switch (eventId)... should I change it and what is the fastest way?

Comment (Kitzunu) [reply src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_twin_valkyr.cpp:280]: Its fine, I was just lazy and put the link.

What I mean is that you shouldnt have the whitespaces within the bracket

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
thanks for your contribution @meowcpp !

**Pull request merged**